src/node/stage/bootstrapping.rs

```diff
@@ -162,6 +162,7 @@ impl Bootstrapping {
 
         let message = Message::single_src(
             &self.node_info.keypair,
+            MIN_AGE,
```

Comment:
When we are relocating, this won't be `MIN_AGE`. You can fetch the correct age from `self.relocate_details`.